lib/twitter-ads/resources/analytics.rb

```diff
@@ -67,8 +67,8 @@ module ClassMethods
 
       def stats(account, ids, metric_groups, opts = {})
         # set default metric values
-        end_time          = opts.fetch(:end_time, (Time.now - Time.now.sec - (60 * Time.now.min)))
-        start_time        = opts.fetch(:start_time, end_time - 604_800) # 7 days ago
+        end_time          = opts.fetch(:end_time, Date.today.to_time)
```

Comment:
This changes the default behavior of the time portion of the date time to be midnight (00:00:00) rather than the current hour. I don't think this is our desired behavior, as we would like usage to continue to pull on a rolling 7-day window by default.